SmartDeviceLink/SDLKeyboardDelegate.h

```diff
@@ -17,10 +17,17 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  This handler is called when you wish to update your autocomplete text in response to the user's input
 
- @param updatedAutocompleteText The new autocomplete text to use
+ @param updatedAutocompleteText The autocomplete results to use
  */
 typedef void(^SDLKeyboardAutocompleteCompletionHandler)(NSString  *_Nullable updatedAutocompleteText);
 
+/**
+ This handler is called when you wish to update your autocomplete text in response to the user's input.
```

Comment:
Should add that a max of 100 items is allowed to be shown at once

Reply:
Fixed